### ISSUE TITLE
how to use tensorwatch to observer loss when train a model  which git clone from res2net

### ISSUE THREAD

**Issue by wangliting-amy** [microsoft/tensorwatch#26]
now I use pytorch and res2net which git from https://github.com/4uiiurz1/pytorch-res2net. when I train this mode i want to observe the loss and accuracy, and also want to observe the structure of network, so I use tensorwatch similar with Using TensorWatch for Deep Learning Training (Food360). But there is no expression.what's the reson?
